src/components/navigation/header/types.ts

```diff
@@ -1,14 +1,5 @@
-export enum UserType {
-  Private = 'private',
-  Professional = 'professional',
+export enum UserTypeExternal {
```

Comment:
What's the idea behind that? There will never be a user with `userType: "guest"` passed to the header or am I missing something?

Reply:
We pass either private, professional or no user at all if not logged in. If no user is passed then we consider it to be Guest: https://github.com/smg-automotive/components-pkg/pull/757/files#:~:text=this.userType%20%3D%20user%20%3F%20user.userType%20%3A%20UserTypeExternal.Guest%3B

Reply:
Ah I see it might be confusing as in legacy they called private users guests....